src/vivarium_gates_nutrition_optimization/components/mortality.py

```diff
@@ -39,30 +50,23 @@ def setup(self, builder: Builder) -> None:
         self.random = self.get_randomness_stream(builder)
         self.clock = builder.time.clock()
         self.mortality_probability = self.get_mortality_probability(builder)
-        self.life_expectancy = self.get_life_expectancy(builder)
 
     ###################
     # Setup Methods   #
     ###################
 
-    # noinspection PyMethodMayBeStatic
-    def get_life_expectancy(self, builder: Builder) -> Union[LookupTable, Pipeline]:
-        life_expectancy_data = builder.data.load(
-            "population.theoretical_minimum_risk_life_expectancy"
-        )
-        return builder.lookup.build_table(life_expectancy_data, parameter_columns=["age"])
-
     def get_mortality_probability(self, builder: Builder):
         probability_data = builder.data.load(
             data_keys.MATERNAL_DISORDERS.MORTALITY_PROBABILITY
         )
-        probability_pipeline_source = builder.lookup.build_table(
-            probability_data, key_columns=["sex"], parameter_columns=["age", "year"]
+        probability_pipeline_source = self.build_lookup_table(
```

Comment:
Is there a reason this isn't in the data_sources config and created automatically by `build_all_lookup_tables`?